omero/users/cli/sessions.rst

```diff
@@ -93,10 +95,10 @@ Some of the options available to the :program:`omero login` command are:
 Multiple sessions
 ^^^^^^^^^^^^^^^^^
 
-Stored sessions can be listed using the :program:`omero sessions list`
+Stored sessions can be listed using the :program:`omero sessions who`
```

Comment:
`sessions list` looks more appropriate to me:
```
$ omero sessions who
Using session for user-5@eel.openmicroscopy.org:4064. Idle timeout: 10 min. Current group: read-write-1
 name   | group        | logged in               | agent          | timeout | hitCount | lastAccessTime
--------+--------------+-------------------------+----------------+---------+----------+---------------------
 user-3 | read-only-1  | ---                     | ---            | ---     | ---      | ---
 root   | read-write-1 | ---                     | ---            | ---     | ---      | ---
 user-3 | read-only-1  | ---                     | ---            | ---     | ---      | ---
 root   | read-write-1 | ---                     | ---            | ---     | ---      | ---
 user-5 | read-write-1 | 2018-08-21 11:04:16 (*) | OMERO.sessions | 10 min. | 141      | 2018-08-21 11:05:25
 root   | read-write-1 | ---                     | ---            | ---     | ---      | ---
 user-5 | read-write-1 | 2018-08-21 11:01:01     | OMERO.sessions | 10 min. | 83       | 2018-08-21 11:01:35
 user-3 | read-only-1  | ---                     | ---            | ---     | ---      | ---
(8 rows)
```
```
$ omero sessions list
 Server                 | User   | Group        | Session                              | Active    | Started
------------------------+--------+--------------+--------------------------------------+-----------+--------------------------
 eel.openmicroscopy.org | user-5 | read-write-1 | <SESSION-ID> | Logged in | Tue Aug 21 11:04:16 2018
(1 row)
```

Reply:
Agreed. `sessions list` is for the local work. `admin sessionlist` is what was deprecated in gh-5811

Reply:
no ref to ``sessionlist`` in the doc